pkg/Library/Designer/designer-layout.js

```diff
@@ -27,8 +27,8 @@ export class DesignerLayout extends DragDrop {
   }
   update() {
     // TODO(sjmiles): the `on-slotchange` doesn't work as expected.
-    setTimeout(() => {this.updateGeometry();}, 100);
-    // this.updateGeometry();
+    //setTimeout(() => this.updateGeometry(), 100);
+    this.updateGeometry();
```

Comment:
for me it often results in the nodes not being rendered at all 🤷 
what am i doing wrong?

Reply:
Race-condition maybe, we should look at it together.